### PR TITLE
Add another host for DoubleClick preconnect.

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

File: ads/_prefetch.js [hunk 1]
```diff
@@ -39,5 +39,6 @@ export const adPreconnect = {
   doubleclick: [
     'https://partner.googleadservices.com',
     'https://securepubads.g.doubleclick.net',
+    'https://tpc.googlesyndication.com',
```
Comment (dvoytenko) [inline ads/_prefetch.js]: Just one clarification. Did we drop our rule to not have trailing comes in arrays?

**Comment by dvoytenko** [comment]
LGTM

File: ads/_prefetch.js (hunk 1 above)
Comment (cramforce) [inline ads/_prefetch.js]: We mandate them! Should at least.

**Comment by ampsauce** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 2036bdc71a51bc67fee06302c47b00bd91160c85
Build details: https://travis-ci.org/ampsauce/amphtml/builds/88466288

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Pull request merged**